src/style.css

```diff
@@ -0,0 +1,355 @@
+}
+
+h2 {
+  text-align: center;
+}
+
+img {
+  width: 370px;
+  height: 500px;
+}
+
+.movie-cards {
+  /* text-align: center; */
+  background-color: beige;
+  padding: 10px;
+  border-radius: 5px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.movie-cards:hover {
+  box-shadow: 0 0 4px #747474ab;
+}
+
+.movie-title {
+  text-align: center;
+  color: #464646;
+  font-weight: 700;
+  text-decoration: none;
+  font-size: 1.5rem;
+}
+
+.movie-info {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
+
+button {
+  width: 100%;
+  padding: 15px;
+  background-color: beige;
+  border: 1px solid brown;
+  border-radius: 10px;
+  font-size: 1.2rem;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: wheat;
+}
+
+button:active {
+  background-color: brown;
+  color: #fff;
+}
+
+.fa {
+  cursor: pointer;
+}
+
+.fa:hover {
+  color: red;
+}
+
+/* For mobile screens */
+@media screen and (max-width: 768px) {
+  .mobile {
+    color: wheat;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 10px;
+    margin-left: 30px;
+  }
+
+  .card-container {
+    display: block;
+    margin: auto;
+  }
+
+  .nav-items {
+    display: none;
+  }
+
+  img {
+    width: 300px;
+    height: 400px;
+  }
+}
+
+.description {
+  display: flex;
+  flex-direction: row;
+  flex: 2;
+  gap: 30px;
+}
+
+.comments {
+  display: flex;
+  gap: 25px;
+}
+
+.mInfo {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  background-color: #f7f7f7;
+}
+
+.D-description {
+  display: block;
+  gap: 20px;
+}
+
+#comment {
+  display: block;
+  outline: 0;
+  width: 15rem;
+  padding: 10px;
+  border: 1px solid brown;
+  border-radius: 4px;
+}
+
+.popup {
+  transition: 200ms ease-in-out;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  width: 80%;
+  height: 99%;
+  padding: 2%;
+  display: flex;
+  background-color: white;
+  z-index: 10;
+}
+
+.summary-tag h1 {
+  color: brown;
+  padding: 0;
+  margin: 0;
+}
+
+.summary-tag {
+  margin-right: 20px;
+  text-align: justify;
+}
+
+.image {
+  width: 250px;
+  height: 370px;
+}
+
+.popup.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.overlay {
+  transition: 200ms ease-in-out;
+  position: fixed;
+  opacity: 0;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.6);
+  pointer-events: none;
+}
+
+.overlay.active {
+  opacity: 1;
+  pointer-events: all;
+}
+
+/* this is where i styled the scroll property for the comments */
+.D-comments {
+  overflow-y: scroll;
+  max-height: 170px;
+  margin: 0;
+  transform: translateX(-40px);
+  cursor: grab;
+}
+
+.D-comments::-webkit-scrollbar {
+  width: 1em;
+}
+
+.D-comments::-webkit-scrollbar-track {
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+
+.D-comments::-webkit-scrollbar-thumb {
+  background-color: grey;
+  outline: 1px solid slategrey;
+  border-radius: 10px;
```

Comment:
Optional 
 although it's not included in the requirements, I will strongly suggest you work on the mobile version of your comment section in the app because it looks disorganized and unattractive to a visitor 👍🏾 

currently your website looks like this 🔽 

![image](https://user-images.githubusercontent.com/60612329/188267404-f9ae04c4-6632-486d-b4dc-73a9d4ddb35b.png)

Reply:
Hello @alexander16108,
if this is Optional then u should've Approved the request, and then Mention it ...
Also, The Number of contribution is equal if check chaabaneAnas commits ull notice that they are some **few** but **Big** ones

Reply:
@ChaabaneAnas 
I understand that he made an equal effort but I'm just implying that he should push more often. and I did Approve the project so you have nothing to worry about